src/chainstate/stacks/db/blocks.rs

```diff
@@ -11649,20 +11893,35 @@ pub mod test {
             account.stx_balance.get_total_balance(),
             1000000000 - (1000 + 2000 + 3000 + 4000 + 5000 + 6000 + 7000 + 8000 + 9000)
         );
+
+        for i in 0..(num_blocks - 1) {
+            let del_addr = del_addrs[i];
+            let result = eval_at_tip(
+                &mut peer,
+                "pox-2",
+                &format!("(get-delegation-info '{})", &del_addr),
+            );
+
+            let data = result.expect_optional().unwrap().expect_tuple().data_map;
+            let delegation_amt = data.get("amount-ustx").cloned().unwrap().expect_u128();
+
+            assert_eq!(delegation_amt, 1000 * (i as u128 + 1));
+        }
     }
 
-    /// Verify that the stacking and transfer operations on the burnchain work as expected in
+    /// Verify that the stacking, transfer, and delegate operations on the burnchain work as expected in
     /// Stacks 2.1.  That is, they're up for consideration in the 6 subsequent sortiitons after
-    /// they are mined (including the one they are in).  This test verifies that TransferSTX
+    /// they are mined (including the one they are in).  This test verifies that TransferSTX & DelegateSTX
     /// operations are only dropped from consideration if there are more than 6 sortitions
     /// between when they are mined and when the next Stacks block is mined.
     #[test]
-    fn test_get_stacking_and_transfer_burn_ops_v210_expiration() {
+    fn test_get_stacking_and_transfer_and_delegate_burn_ops_v210_expiration() {
```

Comment:
does the old test not make sense?

are there any new tests to add?

Reply:
The name of the function being tested changed, so the name of the test changed too.